### PR TITLE
Ip Linux Command

### PR DESCRIPTION
**Pull request by goupvi**
add ip linux coomand for review

### CODE REVIEW

File: ip/ip-command.md [hunk 1]
```diff
@@ -0,0 +1 @@
+### ip
```
Comment (vitaliisili) [inline ip/ip-command.md:1]: change this file to readme.md, fail name is not case sensitive

Comment (vitaliisili) [inline ip/ip-command.md:1]: add information about this command

**Review by vitaliisili — CHANGES_REQUESTED** [review]
this file should be modified

File: ip/ip-command.md (hunk 1 above)
Comment (goupvi) [reply ip/ip-command.md]: file was changed

File: ip/readme.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# ip
+
+*`ip` command in linux*
```
Comment (vitaliisili) [inline ip/readme.md:3]: ok

Comment (vitaliisili) [reply ip/readme.md:3]: thi file should cotain documentation about ip command

File: ip/readme.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
 # ip
 
-*`ip` command in linux*
\ No newline at end of file
+The `ip` command is a powerful tool for configuring network interfaces that any Linux system administrator should know. <br/> 
+It is used to bring interfaces up or down, assign and remove addresses and routes, manage ARP cache, and much more.
+
```
Comment (vitaliisili) [inline ip/readme.md:5]: done

**Review by vitaliisili — APPROVED** [review]
accept

**Pull request merged**